### PR TITLE
Scratch/carousel

### PR DESCRIPTION
**Pull request by cypherpoll-0**

### CODE REVIEW

File: src/app/_app.tsx [hunk 1]
```diff
@@ -95,13 +96,8 @@ export default function Home() {
     }
   }, []);
 
-  const [carouselStyle, setCarouselStyle] = useState<any>({ backgroundImage: "url(/worldmap.svg)", backgroundSize: "contain", backgroundRepeat: "no-repeat" });
+  console.log(openModal)
```
Comment (arijeet-mukherjee) [inline src/app/_app.tsx:99]: remove it

File: src/component/Carousel/index.tsx [hunk 2]
```diff
@@ -135,9 +135,9 @@ const Carousel: React.FC<CarouselProps> = (props: CarouselProps) => {
                             {
                                 className: styles["carousel-content"],
                                 style: {
-                                    width: "100%",
-                                    justifyContent: "space-between",
-                                    userSelect: "none",
+                                    // width: "100%",
```
Comment (arijeet-mukherjee) [inline src/component/Carousel/index.tsx:138]: remove commented code

**Review by arijeet-mukherjee — APPROVED** [review]
LGTM

File: src/component/MobileNavModal/index.tsx [hunk 3]
```diff
@@ -58,15 +62,15 @@ const MobileNavModal: FC<MobileNavModalProps> = ({ list, closeModal, modalState,
                 )}
             </div>
             <div className={styles.btnContainer}>
-            <Button label={headerData.navigation_bar.button.label} hc={handelClick} background='transparent' backgroundOnHover='#F4C9A4' textColor='white' textColorOnHover='black' borderColor='#F4C9A4' fontSize={20}/>
+                <Button label={headerData.navigation_bar.button.label} hc={handelClick} background='transparent' backgroundOnHover='#F4C9A4' textColor='white' textColorOnHover='black' borderColor='#F4C9A4' fontSize={20} />
             </div>
-            <div className={styles.navModalBackground}>
+            {/* <div className={styles.navModalBackground}>
```
[the remark this answers is not in the document]
Comment (cypherpoll-0) [reply src/component/MobileNavModal/index.tsx]: done sir

**Pull request merged**